### PR TITLE
CBL-4123: Add AcceptParentDomainCookies to ReplicatorConfiguration

### PR DESCRIPTION
**Pull request by borrrden**
Bump LiteCore to a higher version that that API changes in

### CODE REVIEW

**Comment by borrrden** [comment]
Sorry about the PR validation, no idea why it's so trash lately.  Need to unwind it and probably redo it.

File: src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs [hunk 1]
```diff
@@ -81,6 +82,17 @@ internal sealed class ReplicatorOptionsDictionary : OptionsDictionary, IDisposab
 
         #region Properties
```
Comment (bmeike) [inline src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs:83]: Not sure what this is for... but it looks like it is probably doing the right thing...

**Review by bmeike — APPROVED** [review]
Is the use of Defaults in the submodule?  I don't se the Property initialize anywhere.

LGTM

File: src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs (hunk 1 above)
Comment (borrrden) [reply src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs:83]: You mean #region?  It's just for code collapsing in IDEs

**Comment by borrrden** [comment]
Hmmm I think actually I blanked completely on the default!  Good catch.

File: src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs (hunk 1 above)
Comment (bmeike) [reply src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs:83]: No.. I meant the whole ReplicatorOptionsDictionary class, distinct from the ReplicatorConfiguration class.  I get it now, though.  There is an analog in Java... it is just written in C in the JNI code

**Pull request merged**